functions/kpi_functions.sql

```diff
@@ -620,3 +620,90 @@ BEGIN
   );
 END
 $$ LANGUAGE plpgsql;
+
+
+-- Unaccompanied Customer Involvement KPI
+-- Employees who have registered more than 50% of their available hours at one specific customer (FG)
+-- and is the ONLY employee to do so for current and preivous month (2 months aggregate)
+
+CREATE OR REPLACE FUNCTION unaccompanied_customer_involvement_kpi(in_from_date date, in_to_date date)
+RETURNS TABLE (from_date date, to_date date, percent double precision) AS 
+$$
+BEGIN
+	RETURN QUERY (
+		SELECT
+			x.from_date,
+			x.to_date,
+			COUNT(ae.employee_count) / t.employee_count :: double precision
+		FROM 
+			month_dates(in_from_date, in_to_date, interval '2' month) x,
```

Comment:
I think one have to start in every month to get this 100% percent correct. 
Example : 
-  0% on ALONE in Jan
- 26% on ALONE in Feb  
- 25% on ALONE in Mar
- 25% on ALONE in Apr
Jan -> Feb => 26 FG on ALONE ,
Mar -> Apr => 50% on ALONE
but:
Feb -> Mar => 51% on ALONE => Oh noes!